packages/cli-helpers/src/lib/installHelpers.ts

```diff
@@ -0,0 +1,30 @@
+import execa from 'execa'
+
+export const addWebPackages = (webPackages: string[], rwVersion: string) => ({
```

Comment:
Should this be part of this PR? I like the refactor, just doesn't seem to have anything to do with Vite

Reply:
I need the addWebPackages function in the setup vite command.

Reply:
Cool. I always just thought of it as an auth specific helper. But of course other commands could use it too! Should we make an auth-cli-helpers package with things only used by the auth setup commands? I guess that kind of ties into the @redwoodjs/paths discussions we've been having. So you don't have to answer. Just something to keep in mind for when we do have the rw/paths conversation next time.

Also don't have to make any code changes here. As I said - I like the refactor 🙂 Just missed that you used it for the vite setup command.